docs/next/guides/building-and-testing/modules.md

```diff
@@ -264,32 +57,32 @@ import {
   PasswordEditor,
   SelectEditor,
   TextEditor,
+  registerEditor, // editors' registering method
 } from 'handsontable/editors';
```

Comment:
Could we also show that the following works:

```
import * from 'handsontable/editors'
```

@budnix if that's correct?

Reply:
Your proposition is not valid ES syntax. If you meant:
```js
import * as editors from 'handsontable/editors';
```
then it's possible but notice that the import does not automatically register the editors. You have to grab the editor and register it using the proper API, for example  `editors.registerEditor(editors.NumericEditor);`. IMO the usage of the API this way is not super elegant. 

If you meant:
```js
import 'handsontable/editors';
```
then this would not work. It wouldn't even work if the editors were registering themselves. The bundler while transpiling the code would remove this import (tree-shaking) causing unexpected results in prod and/or dev builds. That's why there is a requirement to import component and register it manually.

Reply:
Thanks for pointing out the mistake in my line, @budnix.

When someone imports all editors, could they do the following?

```
import * as editors from 'handsontable/editors';
registerAllEditors();
```

If yes, then I think that would be the best way to present `registerAllEditors` in this page.

> The bundler while transpiling the code would remove this import (tree-shaking) causing unexpected results in prod and/or dev builds. That's why there is a requirement to import component and register it manually.

Doesn't this problem apply to `registerAllEditors()`, no matter how the editors are imported? In yes, then we should add a remark about that in the guide.